modules/components/dropdown/tag-input-dropdown.component.ts

```diff
@@ -356,6 +356,9 @@ export class TagInputDropdown {
             if (this.items.length) {
                 this.dropdown.show(this.calculatePosition());
             }
+            else if( !this.showDropdownIfEmpty && this.isVisible ) {
```

Comment:
Hi @matthewerwin can we just adjust the formatting here (as to be consistent with the rest) 

```

}
--
  } else if (!this.showDropdownIfEmpty && this.isVisible) {
}
```